src/main/resources/mcmod.info

```diff
@@ -0,0 +1,16 @@
+[
+{
+  "modid": "dropchevsky",
+  "name": "World Drop",
+  "description": "Lets you set up drops from one dimension to another, at configurable heights.",
+  "version": "${version}",
+  "mcversion": "${mcversion}",
+  "url": "",
+  "updateUrl": "",
+  "authorList": ["Domochevsky and Felina_Lain"],
```

Comment:
This should probably read
`"authorList": ["Domochevsky", "Felina_Lain"],`
for the proper formatting.